masterlist.yaml

```
@@ -18574,3 +18574,24 @@ plugins:
     after:
       - 'BA_KhajiitSpeakRedux_MAIN.esp'
       - 'mjhKhajiitSpeak.esp'
+
+  # Zim's Improved Dremora
+  - name: "Zim's Improved Dremora.esp"
```

Just for consistency, if there's a single qoute in the filename add another to escape it.
```suggestion
  - name: 'Zim''s Improved Dremora.esp'
```